src/Countries/Albania.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Albania extends Country
+{
+    public function countryCode(): string
+    {
+        return 'al';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Viti i Ri' => '01-01',
+            'Dita e Verës' => '03-14',
+            'Dita e Sulltan Nevruzit' => '03-22',
+            'Dita Ndërkombëtare e Punëtorëve' => '05-01',
+            'Dita e Shenjtërimit të Shenjt Terezës' => '09-05',
+            'Dita e Pavarësisë' => '11-28',
+            'Dita e Çlirimit' => '11-29',
+            'Dita Kombëtare e Rinisë' => '12-08',
+            'Krishtlindjet' => '12-25',
```

Comment:
@MirilTerolli `Krishtlindjet` should be `Krishtlindja`

Reply:
@fitimvata it's done.